packages/cozy-harvest-lib/src/services/budget-insight.js

```diff
@@ -200,6 +200,51 @@ export const getBIConnectionId = account => {
   return get(account, 'data.auth.bi.connId')
 }
 
+export const handleWebauthAccount = async ({
+  account,
+  bankId,
+  flow,
+  konnector,
+  client,
+  t
+}) => {
+  let webAuthAccount = clone(account)
+  if (bankId) {
+    set(webAuthAccount, 'auth.bankId', bankId)
+  }
+
+  const connectionId = getWebauthBIConnectionId(webAuthAccount)
+
+  if (connectionId) {
+    logger.info(`Found a webauth connection id: ${connectionId}`)
+    flow.konnector = konnector
+    webAuthAccount = await flow.saveAccount(
+      setBIConnectionId(webAuthAccount, connectionId)
+    )
+
+    await flow.handleFormSubmit({
+      client,
+      account: webAuthAccount,
+      konnector,
+      t
+    })
+  }
+
+  return connectionId
+}
+
+/**
+ * Gets BI webauth connection id which is returned in the account by the stack
+ * via oauth callback url
+ *
+ * @param {io.cozy.accounts} The account content created by the stack
+ *
+ * @return {Integer} Connection Id
```

Comment:
It does not seem to be an Integer here https://github.com/cozy/cozy-libs/pull/1118/commits/9f71f8cdf51eee7fc4954b24296f444cfcf93e97#diff-8f55641483795f28fabc9c030803dd6aR355, can you change it to String ?

Reply:
But this is an integer. This is the result of getWebauthBIConnectionId function